packages/elements-core/src/styles.css

```diff
@@ -2,3 +2,10 @@
 @import '@stoplight/mosaic/themes/default.css';
 
 @import 'core.css';
+
+/* Drawer */
+
+.drawer-container.in.open .left,
```

Comment:
That class name is a bit unfortunate as it's quite generic and as such may conflict with user-defined stylesheets. We should prefix it with `sl-` and tweak the usage of the library if they allow custom CSS class names.

Reply:
Ah. That will require a fix to `mosaic`: https://github.com/stoplightio/mosaic/blob/58f7db3a8c5a5d599ad3c02fd8a3461657a72998/libs/core/src/components/Drawer/Drawer.tsx#L106-L110

Do you want to do that or shall I?